### ISSUE TITLE
no-fallthrough fails to warn about missing break inside braces

### ISSUE THREAD

**Issue by mminer** [eslint/eslint#4652]
With the following code, the `no-fallthrough` rule correctly warns about the missing break statement.

```javascript
let foo = 'bar';

switch (foo) {
  case 'bar':
    console.log('Reached matching case.');

  default:
    console.log('Reached default case.');
}
```

> Expected a "break" statement before "default"

However, when braces surround the case statements, `no-fallthrough` fails to report the problem, even though all cases are indeed executed.

```javascript
let foo = 'bar';

switch (foo) {
  case 'bar': {
    console.log('Reached matching case.');
  }

  default: {
    console.log('Reached default case.');
  }
}
```

Using ESLint v1.10.3.

**Comment by eslintbot** [comment]
@mminer Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by nzakas** [comment]
I believe this is already fixed on master, can you try it out?

**Comment by mminer** [comment]
Yep you're right, the problem is fixed in master. Glad to hear it. I'll go ahead and close this issue.
